kornia/feature/harris.py

```diff
@@ -29,7 +29,11 @@ class CornerHarris(nn.Module):
     :math:`k ∈ [ 0.04 , 0.06 ]`
 
     Args:
-        k (torch.Tensor): the Harris detector free parameter.
+        - k (torch.Tensor): the Harris detector free parameter.
+        - do_blur (bool): perform Gaussian smoothing.
+          Set to True, if use alone, False if on scale pyramid
+        - nms (bool): perform hard non maxima supression
+        - normalize (bool): if True, responce map is divided by max value
```

Comment:
not convinced about the parameters names. Suggest and open to discussion:
`k` -> `factor` (or just `k_factor`)
`do_blur` -> `blur`
`nms` -> `non_maxima_supression`

Reply:
for blur, see below

Reply:
Regarding nms - it is common abbreviation.
Regarding blur - it would be not clear then, what does it mean - if the blur is already there or if it should be added 
Regarding k - I don`t have any preference and actually it was you introducing the "k" :)

Reply:
maybe apply_nms ? I took "k" for the formulation but is zero intuitive :D
We agree with blur -> grads_mode or simply mode?